python-package/lightgbm/sklearn.py

```diff
@@ -193,6 +193,8 @@ def __init__(self, boosting_type="gbdt", num_leaves=31, max_depth=-1,
             Number of parallel threads.
         silent : bool, optional (default=True)
             Whether to print messages while running boosting.
+        importance_type : str, optional (default='split')
+            The type of feature importance to be filled into ``feature_importances_``.
```

Comment:
Please add the description of possible values here.
https://github.com/Microsoft/LightGBM/blob/707ee62ce2a21a7feff55ba78a648fd4c79c65f4/python-package/lightgbm/basic.py#L1875-L1876

Reply:
Added the two options and a reference to the Booster method (to avoid duplication of information). Let me know, if this is not sufficient

Reply:
Good catch about the duplication of information! However, I suppose that users are not so patient to search another docstring (btw, in different module) to get the parameter description.
So, I think it's better just to copy-paste.
BTW, scikit-learn provides even large examples in docstrings to help people understand what's going on right now and right here 😃